### ISSUE TITLE
Adding stratified split option

### ISSUE THREAD

**Issue by Jefidev** [Jefidev/lsfb-dataset#10]
This feature would offer the possibility to ensure that each label in the training set is also represented in the test set. This is only relevant for the LSFB Isol dataset.
